packages/EasyErrorHandler/src/Bridge/Symfony/Commands/AnalyzeErrorCodesCommand.php

```diff
@@ -0,0 +1,112 @@
+<?php
+
+declare(strict_types=1);
+
+namespace EonX\EasyErrorHandler\Bridge\Symfony\Commands;
+
+use EonX\EasyErrorHandler\Bridge\Symfony\Interfaces\ErrorCodes\ErrorCodesProviderInterface;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Helper\Table;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+final class AnalyzeErrorCodesCommand extends Command
```

Comment:
I don't see any changes to the services registration, is it expected that each app will have to manually register this command?

Reply:
@natepage this is a command to run manually on local machines to verify if all the error codes are defined correctly. It's not something that needs to be automated, it's just a helper for developers.

Reply:
I would still register it as a command this way it's really convenient for people to run it

Reply:
Done.